tests/CppWinRTTests.cpp

```diff
@@ -182,6 +184,42 @@ TEST_CASE("CppWinRTTests::VectorToVector", "[cppwinrt]")
     winrt::uninit_apartment();
 }
 
+TEST_CASE("CppWinRTTests::BufferToArrayView", "[cppwinrt]")
+{
+    // Create a buffer of capacity 8 and length 4.
+    auto buffer = winrt::Windows::Storage::Streams::Buffer(8);
+    buffer.Length(4);
+    // Get a Capacity-based int view and set the ints to 256 and 512.
+    {
+        auto view = wil::to_array_view_for_capacity<int32_t>(buffer);
+        REQUIRE(view.size() == 2);
+        view[0] = 256;

```

Comment:
For bit-twiddling I think hex is more understandable than decimal.

Reply:
Got out of the bit twiddling business entirely.